composer.json

```diff
@@ -89,11 +89,12 @@
         "symfony/maker-bundle": "^1.24",
         "symfony/mercure-bundle": "*",
         "symfony/messenger": "^6.4 || ^7.0",
-        "symfony/phpunit-bridge": "^6.4 || ^7.0",
+        "symfony/phpunit-bridge": "^6.4.1 || ^7.0",
```

Comment:
```suggestion
```

Reply:
It will disappear on rebase when added to #6316 :+1: